titan/model.py

```diff
@@ -81,8 +81,11 @@ def __init__(
         random.seed(self.run_seed)
         logging.info(("  FIRST RANDOM CALL {}".format(random.randint(0, 100))))
 
-        logging.info("  Resetting death count")
-        self.deaths: List["ag.Agent"] = []  # Number of death
+        logging.info("  Resetting exit count")
+
+        self.exits: Dict[str, List["ag.Agent"]] = {
+            exit: [] for exit in self.params.classes.exit if exit != "none"
```

Comment:
I think this is checking for a class named "none", but should this instead be if the `exit_type` is none?